Cython/Compiler/Tests/TestCmdLine.py

```diff
@@ -2,6 +2,10 @@
 import sys
 import re
 from unittest import TestCase
+if sys.version_info < (3, 3):
+    from mock import patch, Mock
+else:
+    from unittest.mock import patch, Mock
```

Comment:
I think it's fine to go with a try-except in these cases.
```suggestion
try:
    from unittest.mock import patch, Mock
except ImportError:  # Py2
    from mock import patch, Mock
```

Reply:
Fixed. Thanks!